components/media_player/ui/CardMediaPlayerEPGView.qml

```diff
@@ -90,129 +93,85 @@ Rectangle {
+            property alias epgItemListListView: epgItemListListView
+            Flickable
+            {
+                anchors.top: title.bottom
+                topMargin: 10
 
-                ScrollBar.vertical: ScrollBar {
+                id: fl
+                width: parent.width; height: parent.height-160
+                contentWidth: 15000
+                contentHeight: 8000
+                clip: true
+                boundsBehavior: Flickable.DragAndOvershootBounds
+                ScrollBar.horizontal: ScrollBar {
                     opacity: 0.5
                 }
-
-                header: Component {
-                    Item {
-                        width: parent.width; height: 120
-
-                        Text {
-                            id: title
-                            color: Style.color.text
-                            text: qsTr("EPG") + translateHandler.emptyString
-                            font { family: "Open Sans Bold"; weight: Font.Bold; pixelSize: 40 }
-                            lineHeight: 1
-                            anchors { left: parent.left; leftMargin: 30; top: parent.top; topMargin: 30 }
-                        }
-                    }
+                ScrollBar.vertical: ScrollBar { opacity: 0.5
                 }
+                Repeater
+                {
+                    id:epgItemListListView
+                    model: main.tvchannelListBrowseModel
+                    delegate:epgItemListItem
 
-                populate: Transition {
-                    id: popTransition
-                    SequentialAnimation {
-                        PropertyAction { property: "opacity"; value: 0 }
-                        PauseAnimation { duration: popTransition.ViewTransition.index*100 }
-                        NumberAnimation { properties: "opacity"; from: 0; to: 1; duration: 300; easing.type: Easing.InExpo }
-                    }
                 }
-            }
-
-            Component {
-                id: playListThumbnail
-
-                Item {
-                    id: channelThumbnailItem
-                    width: parent.width-60; height: 80
-                    anchors.horizontalCenter: parent.horizontalCenter
+                Component {
+                    id: epgItemListItem
 
                     Rectangle {
-                        id: tvchannelImage
-                        width: 80; height: 80
-
-                        Image {
-                            source: item_image
-                            anchors.fill: parent
-                            fillMode: Image.PreserveAspectCrop
-                            asynchronous: true
+                        id: epgItemRectangle
+                        x: 20 + item_xCoordinate
+                        y: 10 + item_column * item_height
+                        height: item_height
+                        width: item_width
+                        border.color: "#000000"
```

Comment:
Please use colors from the Style singleton.

Reply:
Resolved